### PR TITLE
Palette: Redness

### PR DESCRIPTION
**Pull request by wbalbo**
**New Contributors:**
Please review [CONTRIBUTING.md](https://github.com/swharden/ScottPlot/blob/master/CONTRIBUTING.md)

**Hacktoberfest Participants:**
Check-out the [Hacktoberfest 2021](https://github.com/ScottPlot/ScottPlot/issues/1274) page

**Purpose:**
* New palette - Redness.
* Issue #1315 
image

### CODE REVIEW

**Review by swharden — APPROVED** [review]

File: src/ScottPlot/Palette.cs [hunk 1]
```diff
@@ -27,7 +27,8 @@ public static class Palette
         public static ScottPlot.Drawing.Palette OneHalfDark => new(new ScottPlot.Drawing.Colorsets.OneHalfDark());
         public static ScottPlot.Drawing.Palette PolarNight => new(new ScottPlot.Drawing.Colorsets.PolarNight());
         public static ScottPlot.Drawing.Palette SnowStorm => new(new ScottPlot.Drawing.Colorsets.Snowstorm());
-        public static ScottPlot.Drawing.Palette xgfs25 => new(new ScottPlot.Drawing.Colorsets.Tsitsulin());
+        public static ScottPlot.Drawing.Palette Redness => new(new ScottPlot.Drawing.Colorsets.Redness());
```
Comment (bclehmann) [inline src/ScottPlot/Palette.cs:30]: If this is supposed to be in alphabetical order then Redness and SnowStorm should be swapped.

**Review by bclehmann — APPROVED** [review]

File: src/ScottPlot/Palette.cs (hunk 1 above)
Comment (wbalbo) [reply src/ScottPlot/Palette.cs:30]: > If this is supposed to be in alphabetical order then Redness and SnowStorm should be swapped.

Makes sense, I will change that in a bit.

Comment (wbalbo) [reply src/ScottPlot/Palette.cs:29]: I reordered the method call but then I needed to get the changes from master, hopefully, will not break anything, let me know, thanks.

Comment (bclehmann) [reply src/ScottPlot/Palette.cs:29]: Looks good to me

**Comment by wbalbo** [comment]
Sorry, forgot to run auto-format this time.

**Pull request merged**